lona/client/dom-updater.js

```diff
@@ -458,16 +463,37 @@ Lona.LonaDomUpdater = function(lona_context, lona_window) {
                     };
 
                     node.removeAttribute(name);
-
                 });
 
                 for(var name in data[0]) {
-                    node.setAttribute(name, data[0][name]);
+                    var value = data[0][name];
+
+                    // properties
+                    if(property_names.indexOf(name) > -1) {
+                        if(name != 'value') {
+                            value = true;
+                        };
+
+                        node[name] = value;
+
+                    // attributes
+                    } else {
+                        node.setAttribute(name, value);
+                    };
                 };
 
             // REMOVE
             } else if(operation == protocol.OPERATION.REMOVE) {
-                node.removeAttribute(data[0]);
+                var name = data[0];
+
+                // properties
+                if(property_names.indexOf(name) > -1) {
+                    node[name] = false;
```

Comment:
```suggestion
                    if(name == 'value') {
                        node[name] = "";
                    } else {
                        node[name] = false;
                    }
```

Reply:
fixed

Reply:
Have you pushed changes?

Reply:
@maratori yep! But to the wrong remote :)